partiql-spi/src/main/kotlin/org/partiql/spi/connector/sql/builtins/AggCountStar.kt

```diff
@@ -1,25 +0,0 @@
-// ktlint-disable filename
```

Comment:
(self-review) `COUNT(*)` builtin function deleted in favor of mapping `COUNT(*)` (`COUNT()` in ast) to `COUNT(1)`